core/uemis-downloader.c

```diff
@@ -1517,6 +1517,10 @@ const char *do_uemis_import(device_data_t *data)
 	if (uemis_mem_status != UEMIS_MEM_OK)
 		result = translate("gettextFromC", ERR_FS_ALMOST_FULL);
 
+	if (data->sync_time) {
```

Comment:
Coding style: no braces around single-line blocks.

Reply:
According to my reading of https://github.com/subsurface/subsurface/blob/master/CODINGSTYLE.md#basic-rules is that there is no prescriptive rule on braces or no braces for single line blocks in the project.
My personal preference is for braces, for maintainability and merge-safety. If this is not desired in the project then `CODINGSTYLE.md` should probably be amended.

Reply:
It's "Kernel Coding Style" which we mostly inherit, except when we don't (e.g. line lengths).

It's not my call and I don't care enough to make CODINGSTYLE.md more explicit on the matter.

Reply:
It's a preference. But a lot of these preferences are driven by people to do most of the work.
Since most of my "Subsurface-time" these days is spent on getting to the point where one day I can transition the maintenance of the backend servers to someone else... I really don't care as much anymore as I used to.

Yes, I like it better if single line blocks don't have brace, but... whatever.